src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -201,20 +214,23 @@ public static void init() {
                 mayorTickScheduled = true;
             }
         });
-        ClientPlayConnectionEvents.JOIN.register(Utils::onClientWorldJoin);
         ClientReceiveMessageEvents.ALLOW_GAME.register(Utils::onChatMessage);
         ClientReceiveMessageEvents.GAME_CANCELED.register(Utils::onChatMessage); // Somehow this works even though onChatMessage returns a boolean
+
+        //Register Mod API stuff
+        HypixelNetworking.registerToEvents(Util.make(new Object2IntOpenHashMap<>(), map -> map.put(LocationUpdateS2CPacket.ID, 1)));
```

Comment:
What's this? It would be nice to see the source code for your mod api.

Reply:
Since I am back on here I just posted it: https://github.com/AzureAaron/hm-api

My library allows for registering to events at mod initialization and it will send the necessary registration packet when receiving the HELLO packet from Hypixel.